pkg/virt-handler/vm.go

```diff
@@ -21,6 +22,18 @@ func NewVMController(listWatcher cache.ListerWatcher, domainManager libvirt.Doma
 				recorder.Event(vm, kubev1.EventTypeWarning, v1.SyncFailed.String(), err.Error())
 				return cache.ErrRequeue{Err: err}
 			}
+
+			restClient, err := kubecli.GetRESTClient()
+			if err != nil {
+				return cache.ErrRequeue{Err: err}
+			}
+			vm.Status.Phase = v1.Succeeded
+			err = restClient.Put().Resource("vms").Body(vm).
+				Name(vm.ObjectMeta.Name).Namespace(kubeapi.NamespaceDefault).Do().Error()
```

Comment:
I am wondering if the state update shouldn't also be done in SyncVM?

Thinking about hotplug: If hotplug succeeds, we will in the future have to update the VM status in some way to indicate that this device is really plugged (mirroring final Domain XML in VM.Status for instance). Of course we could copy our VM object before we pass it to SyncVM and then post it if had been changed by SyncVM in any way, but that sonds a little bit too indirect. Thoughts?